### DISCUSSION TITLE
MINIO Configuration :  MaxConnectionsPerServer

### DISCUSSION THREAD

**Discussion by Yushaoxin** [minio/minio#20624] [Q&A]
现在我的服务器配置是64G运行内存，48线程的固态硬盘服务器，我想要装一个minio，
当前我的设置：
1.MaxConnectionsPerServer=100；
2.MINIO_MAX_IO=100；
3.MINIO_REQUESTS_MAX=10000；

     现在我需要大量并发写入和读取minio数据，所以我在代码中每次请求MINO服务器都都创建了一个minioClient，然后直接读取或下发数据，这样虽然并发很快，但是会导致MINIO服务器的最大连接数，直接占用满了，后续服务无法请求minio。
    我的计划是带代码中设置minioClientPool，我希望创建五个minioClient用于写入minio服务器，3个minioClient用于读取minio服务器，这样每个minioClient需要并发多个请求，因为我也遇到过minio服务器报某个minioClient的请求数量过多，minio服务器拒绝，我现在就想知道大致的一个设置大小。
    我的数据大小和场景：可能同时读取和写入100-200个数据（在内存中），（文件大小1MB-10MB），并且涉及到刚刚写入的数据，就需要即可读取。可能同时请求获取100个数据，此时就可以导致一个minioClient的请求数过多，导致minio服务器拒绝

**Comment by AlexZIX** [comment]
I'm afraid that most peoples here can't understand your question in language different from English and hence can't answer. Can you please translate your question?
